org.d3cod3.Mosaic.yml

```diff
@@ -0,0 +1,438 @@
+app-id: org.d3cod3.Mosaic
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: mosaic
+# rename-icon: accessories-dictionary
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Needs to talk to the network:
+  - --share=network
+  # Needs to save files locally
+  - --filesystem=host
+# cleanup:
+#  - /include
+add-extensions:
+    org.freedesktop.Platform.ffmpeg-full:
+      directory: lib/ffmpeg
+      version: '20.08'
+      add-ld-path: .
+      no-autodownload: true
+      autodelete: false
+cleanup-commands:
+  - mkdir -p ${FLATPAK_DEST}/lib/ffmpeg
+modules:
```

Comment:
Should be reformated to follow https://github.com/flathub/flathub/wiki/YAML-Style-Guide